src/component/button.js

```diff
@@ -212,6 +212,52 @@ SmallButton.propTypes = {
   style: ViewPropTypes.style,
 };
 
+//
+// Small Pill Button
+//
+
+const smallPillStyles = StyleSheet.create({
+  touchable: {
+    paddingTop: 3,
+    paddingBottom: 4,
+    paddingLeft: 15,
+    paddingRight: 15,
+  },
+  text: {
+    fontFamily: 'OpenSans Regular',
```

Comment:
Noticed this text style is used in multiple places, might be good to make a <SmallText> component in text.js?

Reply:
In general it’s good to reuse Text components within larger view components. Buttons are pretty atomic components and have pretty specific styles. So it’s probably more maintainable to specify those here directly.